etcdctl/README.md

```diff
@@ -1174,11 +1159,118 @@ The provided transformer should read until EOF and flush the stdout before exiti
 # Role roleA is revoked from user userA
 ```
 
+### USER DELETE \<user name\>
+
+`user delete` deletes a user.
+
+#### Return value
+
+##### Simple reply
+
+- `User <user name> deleted`. Exit code is zero.
+
+- Error string if failed. Exit code is non-zero.
+
+#### Examples
+
+```bash
+./etcdctl --user=root:123 user delete myuser
+# User myuser deleted
+```
+
+### AUTH \<enable or disable\>
```

Comment:
@xiang90 I had done this deliberately since i thought in order to enable or disable auth first they will need to know about roles and user creation. With the current sequence they can systematically keep doing operations of role related operations, then create users, assign roles to them and then finally enable and disable auth.
But if you fell otherwise let me know will change it as per your suggestion.
Thank you!